packages/@angular/cli/commands/serve.ts

```diff
@@ -20,7 +20,8 @@ export interface ServeTaskOptions extends BuildOptions {
   host?: string;
   proxyConfig?: string;
   liveReload?: boolean;
-  liveReloadClient?: string;
```

Comment:
It's still there, as an alias for `--public-host`. So nothing changes for users of that flag.